runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process.json

```diff
@@ -630,5 +631,8 @@
     "Process.Modeler.ChangeLanguageTitle": "Are you sure you want to change the language?",
     "Process.Modeler.LoseModelerChangesContent": "All unsaved changes process wizard changes will be lost.",
     "Process.Modeler.DialogCancel": "Cancel",
-    "Process.Modeler.DialogConfirm": "Ok"
+    "Process.Modeler.DialogConfirm": "Ok",
+    "Process.Modeler.VariablesPanel.GlobalTag": "Global",
+    "Process.Modeler.VariablesPanel.InputOutputTag": "Input/Output",
```

Comment:
I would suggest already implementing what we establish at the meeting regarding the recent action changes. We don't distinguish if a variable is a global or created via assign action.
so we should have 2 types of variables:
1. Variable
2. Action Output